crates/topos-tce-storage/src/validator/tables.rs

```diff
@@ -71,11 +76,38 @@ impl ValidatorPendingTables {
 
         let db = init_with_cfs(&path, default_options(), cfs)
             .unwrap_or_else(|_| panic!("Cannot open DB at {:?}", path));
+        let pending_pool = DBColumn::reopen(&db, cfs::PENDING_POOL);
+        let next_pending_id = {
+            let cf = pending_pool
+                .rocksdb
+                .cf_handle(cfs::PENDING_POOL)
+                .expect("Cannot get cf handle for pending pool");
```

Comment:
I guess the idea here is that if we can't open the DB then it's better to panic and exit rather than returning an error? Do we have some sort of team-agreement on the kinds of "catastrophic errors" that should stop the node from starting? Maybe something we should add to the style guide?

Reply:
We didn't agree on having such kind of logic but having the node panicking during the storage setup is ok for me, it prevents us to deal with a big error management. And basically, if we can't create the storage, the node will not properly work and should exit. We can discuss if we want a full error management, but for the sake of simplicity, I didn't wanted to introduce it in this PR.

Reply:
I agree with you. I was just wondering if we have any sort of rule around this.